cysync/src/errors/types.ts

```diff
@@ -68,6 +68,8 @@ export enum CysyncError {
   RECEIVE_TXN_CANCEL_FAILED = 'DS_OPTS_1507',
   RECEIVE_TXN_UNKNOWN_ERROR = 'DS_MISC_5502',
 
+  SWAP_TXN_UNKNOWN_ERROR = 'TODO_1234', // TODO: add error code
```

Comment:
Add a code here

Reply:
Can you suggest me the code? Is there any format that we follow? What does the number at the end signify?